### ISSUE TITLE
Missing response key from  /convert-text-with-timestamps endpoint

### ISSUE THREAD

**Issue by Correia-jpv** [team-listnr/text-to-speech-api#3]
I'm requesting /convert-text-with-timestamps but only getting the /convert-text response with the success, url, audioKey, and voiceoverId keys. Missing the timestamps key.

**Comment by geetansh** [comment]
Same issue, I have reported it on Facebook group as well but the post was declined by the admin.
Its been 6 months guys, any update on the issue ?
